examples/nftbillboard-userdata/build-and-test.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/env bash
```

Comment:
I think that a few of these `build-and-test.sh` files are repetitive and can be either replaced by the `testenv-ctl.sh` file in `ethereum-contracts` or if these files are different enough, to have their own `testenv-ctl.sh` file for all example projects-it seems that the projects will all have somewhat similar environment set ups unless there's an example project which exists to illustrate a new way of setting up a test environment (e.g. w/ hardhat network or off a fork, etc.)

Reply:
My one concern here is with using different test environments. I was able to use ganache + truffle within two of the hardhat projects, but may want to use hardhat network specifically for an example project in the future. 

If this is the case, would it still be best to add a` testenv-ctl.sh` file for all example projects?

Reply:
We can leave as it is for this branch. When we have a new testenv-ctl published, we update the examples to use it.

Reply:
Sounds good